README.md

```diff
@@ -102,13 +102,31 @@ yarn ts
 
 2. Create a new branch, and make sure to push it. This is where package version updates, and git tags will be pushed from.
 
-3. Publish all packages to NPM using Lerna. In the future, this will also include automatic changelog creation.
+3. Update each package's changelog:
+
+a) Provide information about what packages to release and what version each package should be bumped to. We follow semver conventions for versioning, so each change will either be major, minor, or patch.
+
+```
+yarn changeset
```

Comment:
Let's move this to the committing section of this README.

Making the changeset should be independent of the release. It is just required in a PR if we have something notable to include in the changelog.

Reply:
👍